### PR TITLE
Revert "Harleestack"

### PR DESCRIPTION
**Pull request by harleehair**
Reverts TheEvergreenStateCollege/upper-division-cs#85

### CODE REVIEW

**Comment by harleehair** [comment]
Continuing from 96 = RemoveLinkedList -hmh

**Comment by harleehair** [comment]
Continuing from 96 = RemoveLinkedList -hmh

**Review by faulkdf — COMMENTED** [review]
reviewed

File: dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/LinkedListWrapper.java [hunk 1]
```diff
@@ -166,11 +166,4 @@ public boolean containsAll(Collection c) {
     public Object[] toArray(Object[] a) {
         return new Object[0];
     }
-
```
Comment (whereismyprozac) [inline dsa-23au/java-dsa/arrays-links/src/main/java/dev/codewithfriends/LinkedListWrapper.java:169]: [Assignment and Co-Creation Week 04](https://github.com/TheEvergreenStateCollege/upper-division-cs/blob/main/dsa-23au/notes/docs/week-04/Assignment-04.md#assignment-and-co-creation-week-04)
Perfect work

**Pull request closed without merging**